demo/dry-run.sh

```diff
@@ -1,6 +1,18 @@
 #!/bin/bash
 
+# Launches a snakefile of your choice in dry run mode (for debugging)
+# Usage: ./dry_run.sh "<snakemake_flags>" <snakefile.smk> <target_rule> 
+# Example: ./dry_run.sh example.smk example_all
+# snakefile.smk The snakefile you want to run
+# target_rule: The name of one of the target rules specified in one of the included Snakefiles
+# snakemake_flags: One or more flags for the snakemake to run, specified inside quotation marks
+
+
 # Default to all targets
-TARGETS=${@:-all}
+snakefile=$1
+TARGETS=${2:-all}
+snakemake_flags=$3
```

Comment:
It looks like you changed the order of the arguments here but you didn't update the Usage

Reply:
Done